src/ctia/task/settings.clj

```diff
@@ -0,0 +1,57 @@
+(ns ctia.task.settings
+  (:require [clojure.tools.cli :refer [parse-opts]]
+            [clojure.tools.logging :as log]
+
+            [schema.core :as s]
+            [clj-momo.lib.es
+             [index :as es-index]
+             [schemas :refer [ESConnState]]]
+            [ctia.stores.es.init :refer [upsert-template!]]
+            [ctia
+             [init :refer [init-store-service! log-properties]]
+             [properties :refer [properties init!]]
+             [store :refer [stores]]]))
+
+(s/defn update-store!
+  "read store properties of given stores and update indices settings."
+  [{:keys [conn index config]
+    {:keys [settings]} :config
+    :as state} :- ESConnState]
+  (try
+    (upsert-template! conn index config)
```

Comment:
The template is already updated when the store is initialised in the `init-es-conn!` fn.